### PR TITLE
Load csv

### PR DESCRIPTION
**Pull request by XiXiaPdx**
This adds songdata.csv to the project and it is being accessed and read from upon startup. Plan is to move data into a system memory database.

### CODE REVIEW

**Comment by XiXiaPdx** [comment]
I am debating storing the records as POJOs in a list/map of some sort or trying a H2 database.  What do you think?

File: src/main/java/spring/songs/App.java [hunk 1]
```diff
@@ -3,13 +3,34 @@
...
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 @SpringBootApplication
 public class App {
 
     public static void main(String[] args) {
-        SpringApplication.run(App.class, args);
+      SpringApplication.run(App.class, args);
+
+      try {
```
Comment (jenirainerpdx) [inline src/main/java/spring/songs/App.java]: pull this code into a service class (this is a simple java class... I use the name "service class" to mean something that lives in a subsection of the Model portion of MVC code organization.  In general, we don't have much "V" (view) in a microservice anymore.  However, the other two elements - M and C - remain good ways to organize code.  Controller = anything that does routing or some sort of way to delegate to underlying model logic.  Model contains primarily POJOs - that reflect the business model of the service or app.  But, it also frequently contains service and utility classes that act on those model/business objects (POJO's).  So, let's try putting that code in a service class and then calling such a class from here.  Alternatively, you don't have to call the service class from main.  You can use the @PostConstruct annotation and that should invoke your service method upon startup of the service.

**Comment by jenirainerpdx** [comment]
> I am debating storing the records as POJOs in a list/map of some sort or trying a H2 database. What do you think?

For now, while we have a low volume of data, pulling POJO's into a map or list is a great idea.  We sometimes call this "poor man's cache".  Ultimately, longer-term, we will want to put the data into some sort of local data store - H2, Derby, Mongo - really your choice on that implementation!

File: src/main/java/spring/songs/App.java (hunk 1 above)
Comment (XiXiaPdx) [reply src/main/java/spring/songs/App.java]: Ahh, Yes! Very helpful explanation and I have gone ahead and created a Model package with service. I probably should create a subdirectory for utils/services too.

Comment (XiXiaPdx) [reply src/main/java/spring/songs/App.java]: I also played around with @PostConstruct and logging shows it is wired up correctly.   

I'm guessing that the magic of the Spring system relies on annotations.  This (annotations in general for Java) is an area I want to understand better...what does this offer the developer/code of application vs if no annotations existed?  I want to tie that understanding into Spring annotations, to peel back the onion a bit and understand whats going on under-the-hood for Spring. 

I've watched some videos on dependency injection/inversion of control for Spring. I believe I understand the concepts of DI/IoC to create code that is loosely coupled and easier to test.

**Pull request merged**